jupiterone/client.py

```diff
@@ -349,3 +411,243 @@ def delete_relationship(self, relationship_id: str = None):
 
         response = self._execute_query(DELETE_RELATIONSHIP, variables=variables)
         return response["data"]["deleteRelationship"]
+
+    def create_integration_instance(self, instance_name: str = None, instance_description: str = None):
+        """Creates a new Custom Integration Instance.
+
+        args:
+            instance_name (str): The "Account name" for integration instance
+            instance_description (str): The "Description" for integration instance
+        """
+        variables = {
+                  "instance": {
+                    "name": instance_name,
+                    "description": instance_description,
+                    "integrationDefinitionId": "8013680b-311a-4c2e-b53b-c8735fd97a5c",
```

Comment:
good callout, this ID is the definitionId of the 'Custom Integration' and the team confirmed this will never change.. I suppose it could be a param to allow for other managed integration instances to be created too

Reply:
added defId as a param with the default being Custom